ci/conda_requirements.txt

```diff
@@ -2,3 +2,4 @@ flake8
 flask
 requests
 pandas
+xlrd
```

Comment:
A very recent change to `xlrd` limits the package to only support `.xls` files, and it will not read `xlsx`.  See this [SO](https://stackoverflow.com/a/65266270) post. Would it be possible to also install "openpyxl"? I'll suggest a change below for parsing